tasks/69_unpack-elements/unpack-elements.js

```diff
@@ -0,0 +1,12 @@
+export const unpackArray = arr => {
+  for (let i = 0; i < arr.length; i++) {
+    while (true) {
+      if (Array.isArray(arr[i])) {
+        arr.splice(i, 1, ...arr[i]);
```

Comment:
Modifying function argument?

Also, why do we need while(true) and then simple if? Couldn't it be simplified?

Also, is a while in a for good practice?